Tests/SwiftDocCTests/Infrastructure/DocumentationContext/DocumentationContextTests.swift

```diff
@@ -2159,11 +2159,35 @@ let expected = """
         XCTAssertNotNil(try context.entity(with: referenceForPath("/Collisions/SharedStruct/iOSVar")))
     }
     
+    func testContextDoesNotLeakReferences() throws {
+        #if os(Linux) || os(Android)
+            throw XCTSkip("'autoreleasepool' is not supported on this platform so this test can not be run reliably.")
+        #endif
+        
+        // Verify there is no pool bucket for the bundle we're about to test
+        XCTAssertNil(ResolvedTopicReference.sharedPool.sync({ $0[#function] }))
```

Comment:
This just duplicates the below test but without the addition of holding onto the `bundle` and `context` since the below test started failing with this change.